tests/cephfs/nfs-ganesha_basics.py

```diff
@@ -47,11 +47,19 @@ def run(ceph_cluster, **kw):
             else:
                 raise CommandFailed("Failed to create nfs cluster")
             # Create cephfs nfs export
-            nfs_client[0].exec_command(
-                sudo=True,
-                cmd=f"ceph nfs export create cephfs {fs_name} {nfs_name} "
-                f"{nfs_export_name} path={path}",
-            )
+            if "5.1" in rhbuild:
```

Comment:
Could you check `if "5.0" in rhbuild` instead of `5.1`?

Reply:
Ok